tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -966,10 +957,6 @@ constexpr bool test() {
         static_assert(is_same_v<decltype(sp_nine.rbegin()), span<int, 9>::reverse_iterator>);
         static_assert(is_same_v<decltype(sp_dyn.rend()), span<int>::reverse_iterator>);
         static_assert(is_same_v<decltype(sp_nine.rend()), span<int, 9>::reverse_iterator>);
-
-        static_assert(noexcept(get<5>(sp_nine)));
-        assert(get<5>(sp_nine) == 60);
-        assert(&get<5>(sp_nine) == begin(sequence) + 5);
     }
 
     {

```

Comment:
I believe that the `auto& [x, y, z] = sp_three;` structured binding test also needs to be removed.

Reply:
Yep, verified it was failing to compile.